terraform/main.tf

```diff
@@ -0,0 +1,124 @@
+provider "aws" {
+  region  = var.aws_region
+  version = "~> 2.32.0"
+}
+
+terraform {
+  backend "s3" {
+  }
+}
+
+# Configure the remote state data source to acquire configuration
+# created through the code in ch-service-terraform/aws-mm-networks.
+data "terraform_remote_state" "networks" {
+  backend = "s3"
+  config = {
+    bucket = var.remote_state_bucket
+    key    = "${var.state_prefix}/${var.deploy_to}/${var.deploy_to}.tfstate"
+    region = var.aws_region
+  }
+}
+locals {
+  vpc_id            = data.terraform_remote_state.networks.outputs.vpc_id
+  application_ids   = data.terraform_remote_state.networks.outputs.application_ids
+  application_cidrs = data.terraform_remote_state.networks.outputs.application_cidrs
+}
+
+# Configure the remote state data source to acquire configuration created
+# through the code in aws-common-infrastructure-terraform/groups/networking.
+data "terraform_remote_state" "networks_common_infra" {
+  backend = "s3"
+  config = {
+    bucket = var.aws_bucket
+    key    = "aws-common-infrastructure-terraform/common/networking.tfstate"
+    region = var.aws_region
+  }
+}
+locals {
```

Comment:
Try to keep all the local variables within the same definition as it's easier to read and debug. I'd also suggest broadly grouping things based on their type, i.e. `locals`, `data`, `module`. We should probably update the style guide to include this

Reply:
TBC...

Reply:
As per chat, OK with the locals blocks split here.